### PR TITLE
Manage Schemas example  

### PR DESCRIPTION
**Pull request by ptyagii**

### CODE REVIEW

File: schemas/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+## Create Schema CR 
+Deploy the ConfigMap resource containing the schema: 
+```
+kubectl apply -f $TUTORIAL_HOME/schema-config.yaml
+```
+
+Deploy the Schema CR: 
+```
+kubectl apply -f $TUTORIAL_HOME/schemas.yaml
```
Comment (MosheBlumbergX) [inline schemas/README.md:73]: need to rename to `$TUTORIAL_HOME/schema.yaml`
```
confluent-kubernetes-examples/schemas/schemas.yaml" does not exist 
```

File: schemas/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+Get the schema for the specified version of the subject `payment-value`: 
+```
+curl http://schemaregistry.confluent.svc.cluster.local:8081/subjects/payment-value/versions/1/schema
+```
+
+## Tear down Confluent Platform
+
+```
+kubectl delete -f $TUTORIAL_HOME/schemas.yaml
```
Comment (MosheBlumbergX) [inline schemas/README.md:107]: `need to rename to $TUTORIAL_HOME/schema.yaml`
```
confluent-kubernetes-examples/schemas/schemas.yaml" does not exist 
```

File: schemas/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+kubectl delete -f $TUTORIAL_HOME/schema-config.yaml
+kubectl delete -f $TUTORIAL_HOME/confluent-platform.yaml
+```
```
Comment (MosheBlumbergX) [inline schemas/README.md:110]: can we add delete of operator please? 
```
helm delete operator
```

**Review by MosheBlumbergX — CHANGES_REQUESTED** [review]
Thanks @ptyagii  , this is really good and been missing for a while now! 
I've been meaning to get to it but happy you did.

I've left few notes.

File: schemas/README.md (hunk 2 above)
Comment (ptyagii) [reply schemas/README.md]: updated the file name

**Pull request merged**